src/Symfony/Bundle/Resources/views/SwaggerUi/index.html.twig

```diff
@@ -73,14 +84,17 @@
             {% set active_ui = app.request.get('ui', 'swagger_ui') %}
             {% if swaggerUiEnabled and active_ui != 'swagger_ui' %}<a href="{{ path('api_doc') }}">Swagger UI</a>{% endif %}
             {% if reDocEnabled and active_ui != 're_doc' %}<a href="{{ path('api_doc', {'ui': 're_doc'}) }}">ReDoc</a>{% endif %}
-            {% if not graphQlEnabled %}<a href="javascript:alert('GraphQL support is not enabled, see https://api-platform.com/docs/core/graphql/')">GraphiQL</a>{% endif %}
+            {% if not graphQlEnabled %}<a href="#" onclick="noGraphQlSupport()">GraphiQL</a>{% endif %}
             {% if graphiQlEnabled %}<a href="{{ path('api_graphql_graphiql') }}">GraphiQL</a>{% endif %}
             {% if graphQlPlaygroundEnabled %}<a href="{{ path('api_graphql_graphql_playground') }}">GraphQL Playground</a>{% endif %}
         </div>
     </div>
 </div>
 
 {% block javascript %}
+    <script type="text/javascript">
+        let noGraphQlSupport = () => alert('GraphQL support is not enabled, see https://api-platform.com/docs/core/graphql/');
```

Comment:
Shouldn't this be put in an external file too?

Reply:
It absolutely could be, I was just trying to limit the scope of changes as much as I could. In this instance, if I override the `javascript` block, and am using, say the https://github.com/nelmio/NelmioSecurityBundle to handle generating nonces, I can still add a nonce to it. The way it was in-lined before would have made that impossible. That being said, if we move it to it's own file we don't have to allow inline JS period, which is nicer.

Reply:
added a commit with the click event moved into a handler in the `init-swagger-us.js` file